src/donation/models/donation.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { UserEntity } from '../../user/models/user.entity';
+
+@Entity('donations')
+export class DonationEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => UserEntity, user => user.donations)
+  user_: UserEntity;
+
+  @Column({ nullable: true })
+  companion_user_id: number;
```

Comment:
How do we handle multiple (e.g. three) companion users in a single donation?

☝️ This could be low prio, so feel free to close if you don't have enough time

Reply:
I've asked about it some time ago and we decided to stay with just one companion user. After all a group of people might go but just one is needed to be a kind of a 'mentor/guide'.